x/airdrop/keeper/keeper_test.go

```diff
@@ -164,6 +166,84 @@ func (s *KeeperTestSuite) TestFeeDrip() {
 	s.Require().Equal(feeCollectorBalance, sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(4040)))
 }
 
+func (s *KeeperTestSuite) TestAddNewFundWithDiffDenom() {
+	amount := sdk.NewInt64Coin("afet", 4000)
```

Comment:
I'd vote for using another denom that `afet` here. This is test and can be anything really, no need to create any confusion with prod tokens.

Reply:
Yep, good call - let me know what you think of "denom" as a placeholder